app/components/date-time-picker.js

```diff
@@ -14,8 +46,20 @@ export default class DateTimePicker extends Component{
       this.minutes = this.date.getMinutes();
     }
   }
+
+  get isoDate() {
+    if (this.date) {
+      let isoString = this.date.toISOString();
+      return isoString.substr(0, isoString.indexOf('T'));
```

Comment:
AuDatePicker / <duet-date-picker> expects the value attribute as ISO-8601 without any time information. Ideally this shouldn't be needed since AuDatePicker can abstract that for us. I think passing the date instances makes more sense.